variables.tf

```diff
@@ -493,6 +493,12 @@ variable "gitab_enable_prom_exporter" {
   default     = false
 }
 
+variable "gitlab_enable_service_ping" {
+  type        = bool
+  description = "Enable Gitlab Service Ping"
+  default     = true
```

Comment:
maybe we can just stick with the gitlab default and keep it false ?

Reply:
gitlab default is "true" as reported here: 
https://docs.gitlab.com/charts/charts/globals.html#general-application-settings

In the Terraform-google-gke-gitlab module was set to  false.